styles/styles.css

```
@@ -31,6 +31,15 @@
   top:10px;
 }
 
+.icon-headings .col svg {
+  display: block;
+  float: left;
+  width: 24px;
+  height: 24px;
+  margin-right: 20px;
+  position: relative;
+  top: 10px;
+}
```

If you're using most of the same styles, it seems better to group them together where possible
```suggestion
.icon-headings .col img,
.icon-headings .col svg {
  display: block;
  float: left;
  width: 24px;
  height: 24px;
  margin-right: 20px;
  position: relative;
  top: 10px;
}

.icon-headings .col svg {
  height: 24px;
}
```